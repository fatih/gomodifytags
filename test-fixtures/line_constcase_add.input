package foo

type foo struct {
	XMLFileID string
	MyExample bool
	MyAnother []string
}
