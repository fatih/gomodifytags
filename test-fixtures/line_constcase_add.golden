package foo

type foo struct {
	XMLFileID string `json:"XML_FILE_ID"`
	MyExample bool   `json:"MY_EXAMPLE"`
	MyAnother []string
}
